examples/lstm_conv.py

```diff
@@ -0,0 +1,136 @@
+from keras.models import Sequential
```

Comment:
This example requires a docstring describing what the example is meant to do, and what the user can expect in terms of results (e.g. time per epoch, final accuracy, etc).